proposal/README.md

```diff
@@ -144,23 +145,24 @@ $ flytectl update projects --id project-x ...
 ```
 
 ## Tasks
- - get
+
```

Comment:
Revert?

Reply:
I didn't get you. Do you want me to revert this change?

Reply:
Yes!